### ISSUE TITLE
Replace missing placeholder files

### ISSUE THREAD

**Issue by mollyaljenkins** [aemsites/vg-roadchoice-com#59]
### Description 
Related to #52 The Brand_name in the Road Choice Canada French website issue is related with the missing placeholder in the placeholder file. Multiple placeholders are missing. 

### Content AC: 
- [ ] Placeholders for RC Canada FR and EN are checked in the below placeholder files

### Placeholder Files:
**Road Choice Canada EN:**
[https://adobe.sharepoint.com/:x:/r/sites/HelixProjects/_layouts/15/Doc.aspx?sourcedoc=%[…]D&file=placeholder.xlsx&action=default&mobileredirect=true](https://adobe.sharepoint.com/:x:/r/sites/HelixProjects/_layouts/15/Doc.aspx?sourcedoc=%7BEEFA3DDF-1462-4234-82A2-24DA733327B1%7D&file=placeholder.xlsx&action=default&mobileredirect=true)
**Road Choice Canada FR:**
[https://adobe.sharepoint.com/:x:/r/sites/HelixProjects/Shared%20Documents/sites/VolvoGr[…]x?d=weefa3ddf1462423482a224da733327b1&csf=1&web=1&e=D4pR6P](https://adobe.sharepoint.com/:x:/r/sites/HelixProjects/Shared%20Documents/sites/VolvoGroup/vg-roadchoice-ca/fr-ca/placeholder.xlsx?d=weefa3ddf1462423482a224da733327b1&csf=1&web=1&e=D4pR6P)

**Comment by Kannan-CM** [comment]
Content updated for the missing Keys

**Comment by Kannan-CM** [comment]
Translation has been updated

**Comment by VladMano** [comment]
Hello!

I believe we have 2 issues here.

- In the Placeholder file links, the first one which is for Canada EN, the sharepoint points to a file which is in French, the same one as for Canada FR.

- I checked the Canada FR placeholder JSON and I found that the following keys are missing:
              -brand_name
              -recommendations_title
              -read_more
This is the same for the one in English. I also attached screenshots.
![image](https://github.com/user-attachments/assets/4deda1b1-5f11-4119-a7d2-c20a6d223b3a)
![image](https://github.com/user-attachments/assets/c6f4381f-fe14-4f10-a1d0-dfe8976f775a)
![image](https://github.com/user-attachments/assets/ccdc01e8-d7df-4874-af9b-2c22621b7ad3)
